_topics/en/pooled-mining.md

```diff
@@ -0,0 +1,216 @@
+  well-known attack against pooled mining.
+
+---
+Hashing a block header will produce a value that can be interpreted as a
+number.  Hashing many unique block headers using double-SHA256 will
+produce a uniform distribution of numbers within the range 0 to
+2<sup>256</sup>-1.  For example, approximately 1% of the numbers will
+have values less than 1% of the range's maximum value.  For a block to
+contain enough proof of work to be valid, its header must hash to a
+value below a _target_ value that is the dynamically set by the
+protocol.  For example, if the target value is 1% of the range maximum,
+each hash of a unique header has a 1% chance of being below the target.
+
+A corollary of the above is that each hash that is below
+the target value will belong to a set of hashes that is below some
+higher target.  For example, if the primary target value is 1%, we would
+expect every hash below that value to have resulted in finding an
+average of 9 other hashes that are all below a secondary target of 10%.
+A header hash below a secondary target is called a _share_ in pooled
+mining.  Only shares below the primary target can produce a valid block,
+but each share demonstrates an amount of proof of work in the same
+proportion as the primary target to the secondary target.  For example,
+if the primary target is 1% and the secondary target is 10%, each share
+proves an amount of work equal to 10% of the average work needed to
+create a block.
+
+The use of shares allows pools to efficiently and equitably track how
+much work is contributed by each member of a pool.  In a simple payout
+scheme, such as _pay per share_ (PPS), a pool may pay Alice 10% of a
+typical block reward for each 10% share she submits.  This allows Alice
+to profit even if she doesn't personally find a hash with the 10x higher
+amount of proof of work needed for a valid block.  If Alice does find
+a proof of work demonstrating that required 10x higher value, a pure PPS
+scheme will still only pay her for one share (e.g. 10%).  PPS schemes
+face various problems in practice, some of which we'll describe later in
+this article, but PPS provides a simple framework for understanding how
+miners can equitably divide the rewards from producing blocks.
+
+Shares are a type of _weak block_.  They need to follow at least two
+rules:
+
+- A share must be either a valid block or what would have been a valid
+  block if it had a header hash below the primary target.
+
+- The coinbase transaction must follow a template provided by the pool.
+  For example, all pools we know of use the coinbase transaction to pay
+  either the pool operator (who later distributes rewards to pool
+  members) or to pay pool members directly.
+
+Pools may impose additional requirements on their members, either
+explicitly or by using a pool protocol that does not give members the
+flexibility to make certain choices.
+
+## Block withholding attacks
```

Comment:
This is fascinating; I wasn't aware of this attack. Would it be worthwhile to add a sentence or two explaining _why_ Mallory might want to do this? What does she get out of it? I'm assuming the answer is that _other miners_ might be Mallory, so this attack should increase the fraction of blocks that they find. But maybe it's worth making that explicit (assuming I'm correct)?